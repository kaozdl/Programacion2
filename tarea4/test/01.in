# linealizacion
linealizacion
insertar_binario (3,x) .
linealizacion
insertar_binario (3,x) (1,y) (5,z) (2,m) (8,c) (-2,b) (12,a) (9,q) .
linealizacion
linealizacion
Fin
