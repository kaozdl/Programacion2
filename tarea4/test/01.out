1>
2>
3>
(3,x)
0
0
4>(3,x)
5>
--(5,z)
-(1,y)
(3,x)
--(9,q)
-(2,m)
--(8,c)
---(-2,b)
----(12,a)
4
2
6>(12,a)(-2,b)(8,c)(2,m)(9,q)(3,x)(1,y)(5,z)
7>(12,a)(-2,b)(8,c)(2,m)(9,q)(3,x)(1,y)(5,z)
8>Fin.
